avm/src/daemon/daemon.rs

```diff
@@ -18,7 +17,7 @@ pub async fn get_ip() -> String {
 
 pub async fn start(agz_file: &str, app_id: &str, deploy_token: &str) {
   let app_id = app_id.to_string();
-  let deploy_token = deploy_token.to_string();
+  let _deploy_token = deploy_token.to_string();
```

Comment:
do we even need this?

Reply:
it will be used later so suppressing the warning for now